var/spack/repos/builtin/packages/model-neocortex/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from spack.pkg.builtin.sim_model import SimModel
+
+
+class ModelNeocortex(SimModel):
```

Comment:
where is this being used? at least I don't find it's usage in this PR (?)

Reply:
It's a module to be avail to the user